_data/projects.yml

```diff
@@ -58,6 +62,8 @@
   repo: uclaacm/teach-la-website
   description: "This website! Built with love and HTML, CSS/SASS, and Jekyll (a static site generator)."
   lead:
+    - name: Vivian
+      hash: vha
     - name: Matt
       hash: mattwang
```

Comment:
Feel free to drop me haha, I'm just an old man now 😞

Reply:
:(

Reply:
lemme check w the leo part also :'(

Reply:
@vivianha534 / @lumisphere902 any thoughts?

Reply:
Feel free to remove me from the current projects listing :+1:

Reply:
ok sounds good!